oioioi/mp/models.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from oioioi.base.utils.deps import check_django_app_dependencies
+
+from oioioi.participants.models import RegistrationModel
+from oioioi.contests.models import Contest
+
+check_django_app_dependencies(__name__, ['oioioi.participants'])
```

Comment:
nice we check for `oioioi.participants`, but why not checking for `oioioi.contests`